conda/recipes/versions.yaml

```
@@ -149,7 +149,7 @@ scipy_version:
 setuptools_version:
   - '>=49,<50'
 spdlog_version:
-  - '>=1.8.5,<2.0.0a0'
+  - '=1.8.5'
```

What about allowing patch fixes?

```suggestion
  - '>=1.8.5,<1.9'
```

---

Funny, that's what I had at first but ended up being conservative just since we know 1.8.5 works. I'm not at all opposed to changing it, though.

---

Would suggest to keep things a bit looser as very tight pinnings tend to cause issues of their own (not getting needed bug fixes without significant effort for example)